_1327/documents/views.py

```diff
@@ -162,14 +164,20 @@ def view(request, title):
 	md = markdown.Markdown(safe_mode='escape', extensions=[TocExtension(baselevel=2), InternalLinksMarkdownExtension(), 'markdown.extensions.abbr', 'markdown.extensions.tables'])
 	text = md.convert(document.text + abbreviation_explanation_markdown())
 
+	can_edit = request.user.has_perm(document.edit_permission_name, document)
```

Comment:
please refactor these lines into a seperate function and call this function wherever you want to check whether a user can edit a document and get the permission overview